bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -115,7 +115,7 @@ async def apply_infraction(
         self,
         ctx: Context,
         infraction: _utils.Infraction,
-        user: UserSnowflake,
+        user: MemberOrUser,
```

Comment:
The following code should be removed from this function

https://github.com/python-discord/bot/blob/8ae21a26e4c74314d0726a37e71877f8566fe6a7/bot/exts/moderation/infraction/_scheduler.py#L168-L174

Reply:
Have removed this locally :+1: